content/docs/0.16.x/reference/files/shipyard/index.md

```diff
@@ -0,0 +1,303 @@
+---
+title: shipyard.yaml
+description: Control orchestation for a Keptn project
+weight: 715
+---
+
+The *shipyard.yaml* file defines the activiies to be performed for a Keptn project
+and the order in which those activities are executed.
+
+Each project must have one, and only one, *shipyard.yaml* file
+which is passed [**keptn create project**](../../cli/commands/keptn_create_project/) command.
+
+* Each *shipyard.yaml* file contains  one or more `stages`, which can be given any name that is meaningful.
+Examples are "dev", "hardening", "production", "remediation".
+A `stage` is a grouping of activities to be executed until the project is deployed and,
+optionally, a "production" stage that defines remediation activities
+that can be executed in response to issues detected on the production site.
+
+* Each `stage` must have one or more `sequences`, which can be given any name that is meaningful.
+A `sequence` defines the `tasks` to be performed and, optionally, an event that triggers that sequence.
+
+The following **Synopsis** shows all the constructions that are supported for a *shipyard.yaml* file
+although most projects only use some of the constructions.
+The requirements are:
+
+## Synopsis
+
+    apiVersion: spec.keptn.sh/0.2.3
+    kind: "Shipyard"
+    metadata:
+      name: "shipyard-<project-name>"
+    spec:
+      stages:
+        - name: "<stagename-1>"
+          sequences:
+          - name: "delivery"
+            tasks:
+            - name: "deployment"
+              properties:
+                deploymentstrategy: "direct" | "blue_greem_service" | "user_managed"
```

Comment:
typo: `blue_green_service` but see notes elsewhere that this `properties` block may not be necessary

Reply:
It is necessary with helm-service, right?  We'll keep working on how to represent this properly